.circleci/config.yml

```diff
@@ -128,26 +136,49 @@ jobs:
             echo "export VERSION_NAME=$VERSION_NAME" >> $BASH_ENV
 
             ./gradlew --stacktrace assembleWordpressJalapenoDebug -PversionName="$VERSION_NAME"
+      - run:
+          name: Build Jetpack APK
+          command: |
+            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
+              PR_NUMBER=$(basename $CIRCLE_PULL_REQUEST)
+              PREFIX="pr-${PR_NUMBER}"
+            else
+              PREFIX="$CIRCLE_BRANCH"
+            fi
+
+            VERSION_NAME="${PREFIX}-build-${CIRCLE_BUILD_NUM}"
+            echo "export VERSION_NAME=$VERSION_NAME" >> $BASH_ENV
```

Comment:
We are exporting this information twice, wouldn't it be better to do a step to calculate and export the version name and then use it in `Build WordPress APK` & `Build Jetpack APK` steps?

Reply:
Yep this would be a cleaner approach here!